src/main/java/ru/yandex/practicum/filmorate/controller/UserController.java

```diff
@@ -55,4 +55,10 @@ public List<User> getUserFriends(@PathVariable("id") Integer userId) {
     public List<User> getCommonFriends(@PathVariable("id") Integer id, @PathVariable("otherId") Integer otherId) {
         return userService.getCommonFriends(id, otherId);
     }
+
+    @DeleteMapping("/{userId}")
+    public void deleteUserById(@PathVariable Integer userId) {
```

Comment:
Аналогично предлагаю добавить имя переменной.
```suggestion
    public void deleteUserById(@PathVariable("userId") Integer userId) {
```